### PR TITLE
eigensoft 8.0.0 (new formula)

### PR DESCRIPTION
**Pull request by alexreg**
- [x] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

File: Formula/eigensoft.rb [hunk 1]
```diff
@@ -0,0 +1,84 @@
+class Eigensoft < Formula
+  # cite Patterson_2006: "https://doi.org/10.1371/journal.pgen.0020190"
+  # cite Price_2006: "https://doi.org/10.1038/ng1847"
+  desc "Eigen tools by Nick Patterson and Alkes Price lab"
+  homepage "https://github.com/DReichLab/EIG"
+  url "https://github.com/DReichLab/EIG/archive/refs/tags/v8.0.0.tar.gz"
+  sha256 "e3459e8ac0134da369910454854eae5c7b261e8816318ccbd2d371b4c6c35741"
+  head "https://github.com/DReichLab/EIG.git"
```
Comment (YoshitakaMo) [inline Formula/eigensoft.rb:8]:
```suggestion
  head "https://github.com/DReichLab/EIG.git", branch: "master"
```

To solve the error "HEAD: Use `branch: "master"` to specify the default branch", add branch info.

File: Formula/eigensoft.rb [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+  depends_on "openblas"
```
Comment (YoshitakaMo) [inline Formula/eigensoft.rb:10]:
```suggestion
  depends_on "gsl"
  depends_on "openblas"
```

To solve the linkage error, add 'depends_on "gsl"'.

File: Formula/eigensoft.rb [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+      assert_match(/^##end of convertf run$/, shell_output("#{bin}/convertf -p par.EIGENSTRAT.PED"))
+      assert_match(Regexp.new(<<~EOS), File.read("example.pedind"))
+        \s+1\s+SAMPLE0 0 0 2 2
+        \s+2\s+SAMPLE1 0 0 1 2
+        \s+3\s+SAMPLE2 0 0 2 1
+        \s+4\s+SAMPLE3 0 0 1 1
+        \s+5\s+SAMPLE4 0 0 2 1
+      EOS
+    end
+
```
Comment (YoshitakaMo) [inline Formula/eigensoft.rb:63]:
```suggestion

  def caveats
    "To generate figures, run `brew install gnuplot`."
  end
   
```

One may require gnuplot to work. But it is not necessary for eigensoft itself. I've described caveats for it.

**Review by YoshitakaMo — CHANGES_REQUESTED** [review]
Thank you for your PR :beer: I've described some suggestions. Please consider them and resubmit to merge.

**Comment by alexreg** [comment]
@YoshitakaMo Thanks for the review. I've incorporated your suggestions now.

**Comment by YoshitakaMo** [comment]
Merged. Thank you, @alexreg !

**Pull request merged**